db/schema.rb

```diff
@@ -214,14 +200,16 @@
     t.text "previous_escape_attempts_details"
     t.string "self_harm"
     t.text "self_harm_details"
-    t.string "date_most_recent_sexual_offence"
     t.string "vulnerable_prisoner"
     t.text "vulnerable_prisoner_details"
     t.string "pnc_warnings"
     t.text "pnc_warnings_details"
-    t.string "date_most_recent_staff_hostage_taker_incident"
-    t.string "date_most_recent_prisoners_hostage_taker_incident"
-    t.string "date_most_recent_public_hostage_taker_incident"
+    t.string "intimidation_prisoners"
+    t.string "intimidation_prisoners_details"
```

Comment:
these should be text fields

t.string "intimidation_prisoners"
t.text "intimidation_prisoners_details"